src/service.ts

```diff
@@ -198,7 +203,7 @@ export class DebugService implements IDebugger.IService {
   getBreakpoints = (): DebugProtocol.SourceBreakpoint[] => {
     return this._model.sidebar.breakpoints.model.breakpoints.map(breakpoint => {
       return {
-        line: breakpoint.line
+        line: breakpoint.line - 1
```

Comment:
This seems to be the reason for https://github.com/jupyterlab/debugger/pull/99#issuecomment-546876211